src/MassTransit/Configuration/Metadata/BusHostInfo.cs

```diff
@@ -41,13 +40,21 @@ public BusHostInfo(bool initialize)
             AssemblyVersion = assemblyName.Version.ToString();
         }
 
+        [JsonInclude]
```

Comment:
So if these are just changed to `{ get; set; }` does that eliminate the need for the attribute?

Reply:
Yes. Without this attribute serializer will generate correct json but deserializer will not fill properties implemented as {get;private set;}